Can_Node_Handler.cpp

```diff
@@ -53,6 +53,12 @@ void Can_Node_Handler::handleMessage(Frame& message) {
     brakeLightOff();
   } else {
     brakeLightOn();
+    if(Dispatcher().isEnabled()){
+      const int16_t brakeExtended = analogBrake << 7;
+      //Do Scaling 
+      const int16_t outputBrake = brakeExtended;
+      writeThrottleMessages(-outputBrake);
```

Comment:
Getting closer, however there's still a bug here. Try thinking about all the different cases that could be triggered and how many times writeThrottleMessages is called in each of them. It should be called exactly once in each case, never more, never less. Currently that is not the case.